### DISCUSSION TITLE
Any way to make a cascaded custom send modification?

### DISCUSSION THREAD

**Discussion by spot2000** [collin80/SavvyCAN#580] [General]
Hi, if I want to make a cascaded custom send modification, is that possible, let my try to explain what Im trying to do.

I want to test a long series of UDS DID on the 0x22 SID, meaning that I want to scan all known IDs for all possible SID.

The SID is a value betwween 0 and 65535 ( 0x0 and 0xFFFF)

But I can only make send modification on one byte at a time, how to make a modification so I can test as below

D3    D4
00     00
00     01
00     02
etc
etc
01    00
01    01
01    02
etc
etc
FF    FA
FF    FB
FF    FC
FF    FD
FF    FE
FF    FF

Can anyone help me with this?

**Comment by collin80** [comment]
Yeah, the custom frame sender is pretty basic and won't really let you easily do that. Now that I fixed the UDS scanner that's probably the way to go. Or, the scripting window could be used too. That requires that you write Javascript to implement the functionality.
